css/src/components/hero.scss

```diff
@@ -100,6 +101,16 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	&.hero-image {
+		// Set direction to prevent image flip for rtl
+
+		direction: $hero-direction;
+
+		.hero-content {
+			direction: $user-text-direction;
```

Comment:
Agreed with Ben that we should continue to keep the text direction in line rtl if that's the preferred reading direction. Is the answer here to modify the hero image and any other relevant containers that are absolutely positioned to not use logical properties? If so, let's be sure we add comments next to them explaining why

Reply:
```css
	.hero-content {
		z-index: $zindex-active;
		padding-block: $hero-padding-sm;
		@include desktop {
			width: $hero-content-fluid-width;
			min-height: $hero-min-height-default;
			padding-block: $hero-padding-default;
			padding-inline-end: $hero-padding-sm; // padding-right
		}
	}
```
	and 
	
```css
		&:not(.hero-image) {
		.hero-content {
   			margin-right: auto; // addition

			@include desktop {
				width: 100%;
				max-width: $hero-content-width;
			}
		}
	}
```
as well as

```css
	&.hero-image::before {
		display: none;
		position: absolute;
		inset-block-start: 0;
		width: $hero-image-width;
		height: 100%;
		background-image: var(--hero-background-image);
		background-repeat: no-repeat;
		background-size: cover;
		content: ' ';
		right: $layout-gap; // here
```

Reply:
I think it's more about DOM order rather than the logical properties. 

Example of replacing the logical properties: https://design.learn.microsoft.com/pulls/599/components/hero.html

It would require adding `position: relative` and `right: some %` to the hero content container, but only for rtl. The image position is also affected by `.gradient-border-right` so it would also require an override of its `border-inline-end-width` property.  

It seems more straightforward to set the correct direction property. I updated the PR to set `hero-content`'s direction to `$user-text-direction` to respect the correct text direction. Its value doesn't change within Atlas so there's no visual change for the Atlas site, but it is set to `rtl` in the rtl stylesheet in Docs-UI.

Reply:
`$user-text-direction` is not something we can rely on the in the future. One goal we should have is to remove that property entirely.

Reply:
Your solution looks great to me, seems to work as intended in a downstream package. Tested with npm link.